uber/site_sections/summary.py

```diff
@@ -147,17 +147,6 @@ def shirt_counts(self, session):
             ]
         }
 
```

Comment:
Noticed this while going through our pages; this was replaced with the newer page that breaks down shirt counts by size and type.